patches/react-native-system-setting+1.7.4.patch

```diff
@@ -0,0 +1,23 @@
+diff --git a/node_modules/react-native-system-setting/ios/RTCSystemSetting.m b/node_modules/react-native-system-setting/ios/RTCSystemSetting.m
+index c94aaad..e2f44a7 100644
+--- a/node_modules/react-native-system-setting/ios/RTCSystemSetting.m
++++ b/node_modules/react-native-system-setting/ios/RTCSystemSetting.m
+@@ -8,7 +8,7 @@
+ 
+ #import "RTCSystemSetting.h"
+ #import <SystemConfiguration/CaptiveNetwork.h>
+-#import <CoreLocation/CoreLocation.h>
++//#import <CoreLocation/CoreLocation.h>
```

Comment:
We can just delete this right?

Reply:
since it's a patch it doesn't really matter. I can delete comments.